doc/setup.md

```diff
@@ -197,15 +199,7 @@ SYSTEM_PASSWORD=mutualaid
 
 then run the bootstrapping process.
 
-### Running tests
-
-```
-bin/dev/test
-```
-
-Runs the tests inside the context of the `app` container
-
-Analogous to: `bin/test`
+**BEWARE**  This script is intended to be a quick way to start from noting and get a working instance of the application - suitable for development - up and running. It is also appropriate if you should find yourself at a total loss and want to start over with your development environment. This destroys and recreates the volumes for the various containers - IE databases, gem caches, yarn caches will be destroyed - which can be costly in wall clock time.
```

Comment:
start from "nothing" ?

Reply:
Maybe 

"It is also appropriate if you want to start over with your development environment (like, if the data is now outdated due to code changes, it's acting wonky in a way you can't explain, etc).

Reply:
by "wall clock time", do you mean, "which will take several minutes to run" ?

Reply:
It always sounds good enough in my head, but that doesn't mean it makes sense to another human being. ;-) I appreciate the suggestions!